near/src/validator_manager.rs

```diff
@@ -71,10 +71,10 @@ fn find_threshold(stakes: &[Balance], num_seats: u64) -> Result<Balance, Validat
             break Ok(left);
         }
         let mid = (left + right) / 2;
-        let mut current_sum = 0u64;
+        let mut current_sum: Balance = 0;
         for item in stakes.iter() {
-            current_sum += item / mid;
-            if current_sum >= num_seats {
+            current_sum += *item / mid;
+            if current_sum >= num_seats.into() {
```

Comment:
This looks like it can fail in runtime, especially if we are going to have validators with a large balance that exceeds u64.

Reply:
In which case? 
Sum will always be below u128 (as it's maximum our circulating supply). Num seats gets coverted to u128 here.

Reply:
I see. I confused it with u128 -> u64 conversion.